src/platform/src/core/utils/proxyClient.js

```
@@ -10,18 +15,46 @@ import axios from 'axios';
 export const createProxyHandler = (options = {}) => {
   const { requiresAuth = false, requiresApiToken = false } = options;
 
-  return async (req, res) => {
-    // Extract the target path from the request
-    const { path, ...queryParams } = req.query;
+  return async (req, context) => {
+    // Handle both App Router and Pages Router formats
+    let path, queryParams, res;
+
+    if (context && context.params) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)